lib/github-client.js

```diff
@@ -10,6 +10,7 @@ function GithubClient (config) {
   this.base = config.base || 'production'
   this.endpoint = config.endpoint || 'https://api.github.com'
   this.fetchAllPRs = config.fetchAllPRs || false
+  this.prNo = config.prNo || -1
```

Comment:
Is the intention of using `-1` to show that there is no PR number?

If so, use `null` or `undefined`.